auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/FetchUserAuditController.kt

```diff
@@ -34,14 +34,19 @@ class FetchUserAuditController(
     private suspend fun ApplicationCall.getUserAuditJson() {
         val session = principal<OAuthSession>()!!
         val page = parameters["page"]?.toInt()
+        // TODO DT-1022 - send GUID instead of CN
```

Comment:
Yeah we could remove the actual user CN from this JSON response. I think we don't want to be doing potentially lots of AD lookups to get all the editing users, it would be alright for here, but not for the CSV where there could be a lot of unique users.